_posts/2015-11-03-what-you-can-learn-when-you-pair-and-collaborate.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+name: what-you-can-learn-when-you-pair-and-collaborate
+title: 'What you can learn when you pair and collaborate'
+date: 2015-11-03 09:00:00 +00:00
+author: Mani Sarkar
+image:
+    src: /assets/img/custom/blog/why-collaborate.png
+    attribution:
+        text: None of us is as smart as all of us - Kenneth H. Blanchard
+        href: https://kostiukg.files.wordpress.com/2014/12/why-collaborate.png
+---
+
+#### Introduction
+
+It's been over a month since I have joined 'Codurance', a formidable force, and its about time that I share my experiences with my fellow mates and the community around me.
```

Comment:
Remove the `'` quotes around Codurance. I assume you want to insist on the company name, but I don't think it's done this way in English. Better to put the name in bold for example.